### PR TITLE
Update error email

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jjbennett** [comment]
**lurch: attach W-031884

**Comment by jjbennett** [comment]
**lurch attach: W-032354

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008Llb7QAC" target="_blank">W-031884</a>

File: src/classes/InstallScript.cls [hunk 1]
```diff
@@ -29,9 +29,10 @@
 */
 public without sharing class InstallScript implements InstallHandler {
     private static final String EMAIL_REPLY_TO = 'donotreply@salesforce.org';
-    private static final String EMAIL_TO = 'jennifer.bennett+errors@salesforce.com';
+    private static final String EMAIL_TO = 'errors@salesforcefoundation.org';
     private static final String EMAIL_SUBJECT = 'V4S Install Errors';
     private static final String EMAIL_SENDER = 'V4S Install Script';
+    private static final String EMAIL_MESSAGE = 'Org Id: {0} encountered the following error: {1}';
```
Comment (jjbennett) [inline src/classes/InstallScript.cls]: @katherinemitchell - This email message will get sent to the lead developers when an error occurs attempting to send the telemetry and will not be seen by the end user. Let me know if you think the wording should be changed. The {0} will be swapped out with the org Id that encountered the error and the {1} will be whatever error is caught. Thanks!

Comment (katherinemitchell) [reply src/classes/InstallScript.cls]: 'Org ID {0} encountered this error: {1}'

Comment (katherinemitchell) [reply src/classes/InstallScript.cls]: @jjbennett

**Comment by jjbennett** [comment]
@kselvocki - I just tagged you on this work item but realized this one is actually very difficult to test. An error would need to be thrown while doing an upgrade to the package and when calling the feature parameter feature. The only way I know of that could work is if we max out an api limit and then attempt the upgrade but not sure if we want to go that far... @lmeerkatz @scottwarren-sfdo  thoughts?

**Comment by jjbennett** [comment]
**lurch detach: W-031884

File: src/classes/InstallScript.cls [hunk 2]
```diff
@@ -80,7 +81,12 @@ public without sharing class InstallScript implements InstallHandler {
 
         } catch (Exception ex) {
             // Email error and allow the install to continue.
-            sendError(ex.getMessage());
+            sendError(
+                String.format(
+                    EMAIL_MESSAGE,
+                    new List<String>{ context.organizationId(), ex.getMessage() }
```
Comment (lmeerkatz) [inline src/classes/InstallScript.cls]: Would org Id or message ever be null, and if so would this still work?

**Comment by lmeerkatz** [comment]
@jjbennett @kselvocki I don't think there's anything meaningful that's worth QA'ing here. Thanks for checking!

**Comment by jjbennett** [comment]
**lurch: detach

**Comment by LurchTheButler** [comment]
Detached <a href="https://foundation.my.salesforce.com/a2x1E000008Llb7QAC" target="_blank">W-031884</a>

**Comment by jjbennett** [comment]
**lurch: attach W-032354

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008LmDkQAK" target="_blank">W-032354</a>

File: src/classes/InstallScript.cls (hunk 2 above)
Comment (jjbennett) [reply src/classes/InstallScript.cls]: Neither should ever be null but if we passed in null we would get the string formatted with nulls as strings, example:
`System.debug(String.format('test {0} and {1}', new List<String>{ null, null }));`
Results in: 'test null and null'

**Pull request merged**